src/client/theme-default/components/VPFlyout.vue

```diff
@@ -105,19 +105,13 @@ function onBlur() {
 .text {
   display: flex;
   align-items: center;
-  line-height: var(--vp-nav-height-mobile);
```

Comment:
Button styles already center the button's contents, so we don't need to use the big line-height trick here anyway.